src/platform_impl/linux/x11/event_processor.rs

```diff
@@ -158,6 +158,22 @@ impl<T: 'static> EventProcessor<T> {
 
         let event_type = xev.get_type();
         match event_type {
+            ffi::MappingNotify => {
```

Comment:
This part is suspicious, it was in our `neovide/winit`, but not in yours code. Should i remove it?

Reply:
Hmm, no, it was not in our neovide fork too.

Maybe it got here from some unresolved cherry-pick conflicts, but i'll wait for your feedback before removing it.